config.toml

```diff
@@ -57,9 +57,7 @@ link = "https://unpkg.com/lunr/lunr.js"
 ## English Navigation
 [[Languages.en.menu.main]]
 identifier = 'Ensign'
-pageRef = 'ensign'
```

Comment:
The link to `/ensign` was removed from `Ensign` in the nav bar so that users may only be navigated to the page if they click `Ensign for Data Teams` in the drop-down menu.